### ISSUE TITLE
Incorrect detection length in v2 user manual

### ISSUE THREAD

**Issue by vovodroid** [bigtreetech/smart-filament-detection-module#67]
User manual says `Detection Length: 2.88 mm`. I performed real test counting state change events in Marlin code. There are 670 pulses per 1000 mm, resulting in about 1.4925 mm/step.

**Comment by Simon-CR** [comment]
I vibed coded some changes in klipper to allow the filament_motion_sensor.py to be used more like an odometer and be able to use the values (and reset to 0 at will) in macro.  My very poor measuring landed me at mm_per_pulse ≈ 1.56 which is pretty close to the number you got.  My desire was to make a spool usage calculator, but I'm now not sure how accurate this can be.

if anyone is interested, attached is the .md file of the work I've done to it.  

[sfs_encoder_pulses.md](https://github.com/user-attachments/files/25214854/sfs_encoder_pulses.md)

**Comment by vovodroid** [comment]
> ≈ 1.56

My further measurement gave me 681 per meter - 1.47

**Comment by Simon-CR** [comment]
yeah .. I'll use the fact that I can now see pulses to help with my load/unload macros, but not as a proper "length" reference
